queries/go/refactoring.scm

```diff
@@ -1,12 +1,16 @@
 ;; Grabs all the local variable declarations.  This is useful for scope
 ;; variable passing.  Which variables do we need to pass to the extracted
 ;; function?
-((short_var_declaration
- (expression_list
-   (identifier) @definition.local_name)))
-((var_declaration
- (var_spec
-   (identifier) @definition.local_name)))
+(short_var_declaration
```

Comment:
@teddylear this will eventually go away.  I do want to push toward nodes, as they are testable and not so blackbox like as these files.